.github/workflows/go-test.yaml

```diff
@@ -225,6 +225,23 @@ jobs:
           kubectl -n test-cluster scale deployment rook-ceph-osd-0 --replicas 0
           kubectl rook-ceph --operator-namespace test-operator -n test-cluster rook purge-osd 0 --force
 
+      - name: List CRDS
+        run: |
+          set -ex
+          kubectl -n test-cluster get all
+
+      - name: Destroy Cluster (removing CRDS)
+        env:
+          ROOK_PLUGIN_SKIP_PROMPTS: true
+        run: |
+          set -ex
+          kubectl rook-ceph --operator-namespace test-operator -n test-cluster destroy-cluster
+
+      - name: Validate destroyed CRDS
```

Comment:
Other commands tested in the CI usually have one CI step. How about we combine the destroy and validate steps to a single step to follow the same pattern?